templates/license_header_APACHE.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2017 Fraunhofer Institute for Manufacturing Engineering and Automation (IPA)
```

Comment:
For discussion: should we put in the `2017` or something like `<YEAR>`?

Anyways, this will probably not get adapted :-S